platformio.ini

```diff
@@ -28,7 +28,7 @@ extra_configs =
   platformio_esp32c6_envs.ini
 
 ;default_envs = normal_ESP32_4M
-default_envs = max_ESP32_16M8M_LittleFS
+default_envs = normal_ESP32c6_4M316k_LittleFS_CDC
```

Comment:
Is this change intentional?

Reply:
Nope, was a test here to get the syntax highlighting being based on the C6 config.